src/lib/mod.rs

```diff
@@ -0,0 +1,30 @@
+pub mod appconfig;
+pub mod bigquery;
+pub mod check_subscriptions;
+pub mod cj;
+pub mod controllers;
+pub mod models;
+pub mod settings;
+
+#[cfg(test)]
+pub mod test_utils {
+    use fake::{Fake, StringFaker};
+
+    use crate::settings::Settings;
+
+    pub fn random_simple_ascii_string() -> String {
+        const ASCII: &str = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ._-";
+        let f = StringFaker::with(Vec::from(ASCII), 8..90);
+        f.fake()
+    }
+
+    pub fn empty_settings() -> Settings {
+        Settings {
+            host: "_".to_string(),
```

Comment:
If you make `let empty_string = "_".to_string()` and try assigning empty_string to each property it won't compile with the error

```
        let empty_string = "_".to_string();
   |             ------------ move occurs because `empty_string` has type `std::string::String`, which does not implement the `Copy` trait
...
27 |             environment: empty_string,
   |                          ------------ value moved here
28 |             gcp_project: empty_string,
   |                          ^^^^^^^^^^^^ value used here after move
```

I'm not sure making a function empty_string() is any cleaner than what I already have so leaving as is.